docs/api/validate.rst

```diff
@@ -69,8 +69,6 @@ for an introduction to this API and a list of examples.
     :func:`validate()` is implemented using the stdlib's :func:`functools.singledispatch` decorator, where more specific
     schemas overload the default implementation with more validation logic.
 
-    ----
-
```

Comment:
I don't get why this is suddenly an issue according to their dtd. Stupid and sad...